app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/create' do
+    if params[:content] != ""
+      @tweet = Tweet.create(content: params[:content])
+      @tweet.user_id = current_user.id
+      @tweet.user = current_user
+      @tweet.save
+      redirect "tweets/#{@tweet.id}/show"
+    else
+      redirect "/tweets/new"
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    if logged_in?
+      @tweet = Tweet.find_by(id: params[:id])
+      erb :'/tweets/edit_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets/:id/edit' do
+    if logged_in? && params[:tweet] != ""
+      @tweet = Tweet.find_by(id: params[:id])
```

Comment:
I would contend that you still want to double check the user's association to the tweet here. There are other ways to submit a PATCH/POST request than going through an HTML form...